cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/ConsoleSettingsHttpHandler.java

```diff
@@ -107,12 +107,54 @@ public void set(FullHttpRequest request, HttpResponder responder) throws Excepti
     //Config Properties : Map (Key = CONFIG_PROPERTY, Value = Serialized JSON string of properties)
     //User Settings configurations are stored under empty NAMESPACE.
     Map<String, String> propMap = ImmutableMap.of(CONFIG_PROPERTY, data);
+    LOG.info("Creating prop map " + propMap);
     String userId = Objects.firstNonNull(SecurityRequestContext.getUserId(), "");
     Config userConfig = new Config(userId, propMap);
     store.put(userConfig);
     responder.sendStatus(HttpResponseStatus.OK);
   }
 
+  @Path("/")
+  @POST
+  @AuditPolicy(AuditDetail.REQUEST_BODY)
+  public void add(FullHttpRequest request, HttpResponder responder) throws Exception {
+    String data = request.content().toString(StandardCharsets.UTF_8);
```

Comment:
This is checked further down the line when the request body is converted into a JsonObject. In case the body is empty an error is thrown - 400: Invalid JSON in body